btrdb/utils/general.py

```diff
@@ -74,6 +74,48 @@ def from_nanoseconds(cls, nsec):
                 break
         return cls(pos)
 
+    def for_aligned_windows(self, start, end):
+        """
+        Returns the aligned_windows's fist and last timestamps, as well as the number of windows,
+        based on a given pointwidth set.
+
+        Parameters
+        ----------
+        start : int or datetime like object
+            The start time in nanoseconds for the range to be queried. (see
+            :func:`btrdb.utils.timez.to_nanoseconds` for valid input types)
+
+        end : int or datetime like object
+            The end time in nanoseconds for the range to be queried. (see
+            :func:`btrdb.utils.timez.to_nanoseconds` for valid input types)
+
+
+        Returns
+        -------
+        aligned_start: int
+            First timestamp would be returned by `aligned_windows` that is inclusive of specified start
+            timestamp.
+        aligned_end: int
+            Last timestamp would be returned by `aligned_windows` that is inclusive of specified start
```

Comment:
```suggestion
            Last timestamp would be returned by `aligned_windows` that is inclusive of specified end
```

Just to confirm, this will be `inclusive` or `exclusive` of the specified end time?